converters/toZigbee.js

```diff
@@ -431,9 +440,10 @@ const converters = {
                 const result = await converters.light_color.convertSet(entity, key, value, meta);
                 if (result.state) {
                     result.state.color_temp = utils.xyToMireds(result.state.color.x, result.state.color.y);
+                } else if (value.hasOwnProperty('hue') && value.hasOwnProperty('saturation')) {
+                    result.state = {color: {h: value.hue / 65535 * 360, s: value.saturation / 2.55}};
+                    return result;
                 }
-
-                return result;
```

Comment:
Why has this been removed? I think it still required for https://github.com/Koenkk/zigbee-herdsman-converters/pull/841/files#diff-3e0c275bd41af699153bf59d8dda4a2aR442

Reply:
It shouldn't have been, I added it back now!